packages/paste-website/src/components/tokens-list/TokensExample.tsx

```diff
@@ -17,10 +17,31 @@ export const ColorBox: React.FC<BackgroundColor> = ({backgroundColor}) => {
   return <Absolute backgroundColor={backgroundColor} padding="space50" preset="fill" />;
 };
 
-type BorderBoxProps = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
-export const BorderBox: React.FC<BorderBoxProps> = ({borderColor, borderWidth}) => {
+type BorderBoxProp = Pick<BoxProps, 'borderColor' | 'borderWidth'>;
+interface BorderBoxProps extends BorderBoxProp {
```

Comment:
Maybe just extend `Pick<BoxProps, 'borderColor' | 'borderWidth'>` rather than creating a new type and extending that